projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cdc/cdc.ts

```diff
@@ -97,3 +111,32 @@ export function verifyProfileUpdateSuccess() {
     user.fullName + UPDATED_NAME
   );
 }
+
+export function getSampleB2BUser() {
+  return {
+    titleCode: 'Mr',
+    firstName: 'Spartacus',
+    lastName: 'B2BAdmin',
+    fullName: 'Spartacus B2BAdmin',
+    password: 'Password123.',
+    email: 'spartacusb2b@hybris.com',
+    phone: '555 555 555',
+    address: {
+      city: 'Los Angeles',
+      line1: '1111 S Figueroa St',
+      line2: 'US-CA',
+      country: 'United States',
+      state: 'California',
+      postal: '90015',
+    },
+    payment: {
+      card: 'Visa',
+      number: '4111111111111111',
+      expires: {
+        month: '12',
+        year: '2027',
+      },
+      cvv: '123',
+    },
+  };
+}
```

Comment:
In this or in a different PR (not urgent) let's unify the places where we define the test user details for `spartacusb2b@hybris.com`.

Anjana already added it to develop here:
https://github.com/SAP/spartacus/blob/ec2e4b7f3e8f4b4cd134960ec4089885288ba502/projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cdc/cdc.ts#L19-L24

Reply:
yes @Platonn,

However, I am enriching the user with additional address info needed for e2es.